examples/scripts/observable_paulistring.py

```diff
@@ -0,0 +1,42 @@
+"""Example 2: Expectation value of an observable"""
+
+import numpy as np
+from mpqp.gates import H, Rx
+from mpqp import QCircuit
+from mpqp.execution import run
+from mpqp.measures import ExpectationMeasure, Observable
+from mpqp.execution.devices import ATOSDevice, IBMDevice, AWSDevice
+
+from mpqp.core.instruction.measurement.pauli_string import I, X, Y, Z, PauliString
+
+obs = Observable(
+    np.array(
+        [
+            [2, 0, 0, 0],
+            [0, 0, 0, 0],
+            [0, 0, 2, 0],
+            [0, 0, 0, 0],
+        ],
+        dtype=float,
+    )
+)
+
+ps_obs = Observable(1 * I @ Z + 1 * I @ I)
+
+obs = ps_obs.to_matrix()
+
+print(obs)
+print(ps_obs)
```

Comment:
Printing the observable defined by a matrix will only print the matrix, while printing the observable defined by pauliString will be printed "Observable(1*I@I + 1*I@Z)". Either we put Observable in front of both, to indicate they are indeed observables, otherwise only the representation , to be coherent.
I would "Observable" for both, since the matrix can represent other objects (gates, ...)